app/packages/looker/src/util.ts

```diff
@@ -519,3 +519,9 @@ export const getMimeType = (sample: any) => {
 
 export const isFloatArray = (arr) =>
   arr instanceof Float32Array || arr instanceof Float64Array;
+
+export const getArrayBufferFromUrl = async (url: string) => {
+  const data = await fetch(url);
```

Comment:
`getFetchFunction` in `@fiftyone/utilities` will need to be used here to support all environments.

Reply:
Makes sense, addressed in the next commit! Thanks for the review!